src/schemas/ep00/FlatbufferReader.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "../../FlatbufferReader.h"
+#include <flatbuffers/flatbuffers.h>
```

Comment:
```suggestion
#include "flatbuffers/flatbuffers.h"
```
I think that seems more correct, but happy to be overruled

Reply:
`flatbuffers.h` is an external library header, so I think `<>` is fine in this case, but I agree with you on some of the other cases you've pointed out.